demos/tab-indicator.scss

```diff
@@ -12,69 +12,74 @@
   @include mdc-elevation(1);
 
   display: flex;
-  align-items: center;
-  width: 301px;
+  width: 400px;
   height: 50px;
   margin: 2rem 0;
 }
 
 .demo-item {
-  position: relative;
-  flex: 1 0 auto;
-  width: 150px;
-  max-width: 150px;
+  @include mdc-tab-indicator-surface;
```

Comment:
so will this `mdc-tab-indicator-surface` go on `mdc-tab`?

Reply:
Exactly